packages/squiggle-lang/src/fr/calculator.ts

```diff
@@ -29,7 +29,7 @@ export const library = [
             ["fn", frLambda],
             ["title", frOptional(frString)],
             ["description", frOptional(frString)],
-            ["inputs", frArray(frInput)],
+            ["inputs", frOptional(frArray(frInput))],
```

Comment:
Note that we check separately to ensure that the ``inputs`` length is the same the ``fn`` length, so I think this is less needed.